sidecar/sokuji_sidecar/llama_runtime.py

```
@@ -151,15 +153,24 @@ def _run_probe(rel: str, workdir: str) -> str:
 
+_METAL_CONFIGS = ("m1", "m2", "m3", "m4", "m5")   # newest LAST (fallback pick)
+
+
 def _metal_config() -> str:
-    """Apple chip family from the CPU brand string ('Apple M4 Pro' -> 'm4')."""
+    """Apple chip family from the CPU brand string ('Apple M4 Pro' -> 'm4').
+    An unknown/newer chip degrades to the newest known bucket config with a
+    stderr warning instead of raising — newer Apple GPUs run older Metal
+    binaries fine, and refusing to install would brick every future chip
+    until we ship an update (D11)."""
     brand = subprocess.run(["sysctl", "-n", "machdep.cpu.brand_string"],
```

Ruff (0.15.20)

[error] Starting a process with a partial executable path

(S607)